clients/hadoopfs/src/main/java/io/lakefs/LakeFSFileSystem.java

```diff
@@ -446,6 +449,8 @@ public boolean exists(Path path) throws IOException {
         ObjectsApi objects = lfsClient.getObjects();
         ObjectLocation objectLoc = pathToObjectLocation(path);
         try {
+            LOG.debug(
+                    "[DEBUG] loc " + objectLoc.getRepository() + " " + objectLoc.getRef() + " " + objectLoc.getPath());
```

Comment:
```suggestion
            LOG.debug("loc {} {} {}", objectLoc.getRepository(), objectLoc.getRef(), objectLoc.getPath());
```

Reply:
True.  Actually not useful in the longer term, so removing entirely.